website/src/react-native/versions.js

```diff
@@ -14,7 +14,9 @@ var Site = require('Site');
 
```

Comment:
My editor automatically ran Prettier on this page, which explains all the non-copy changes.

Happy to undo these if necessary.